src/lib.rs

```diff
@@ -6,4 +6,5 @@ pub mod scheduler;
 pub mod http;
 pub mod args;
 pub mod lnd;
+pub mod mempool;
```

Comment:
maybe replace `mempool` with `recommend` or something less specific (we will likely want other feeds for these recommendations, im working on one now and will PR to your PR.)